extensions-contrib/kubernetes-overlord-extensions/pom.xml

```diff
@@ -34,6 +34,15 @@
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
+  <dependencyManagement>
```

Comment:
I'm confused about this one since the other kubernetes extensions can use 2.0. Can we add a comment as to which dependency is holding us back here?

Reply:
kuberenetes-overlord-extensions uses different Java library for kubernetes: io.fabric8.kubernetes-client-api 

core kubernetes extensions uses: io.kubernetes.java client